libkbfs/mdserver_remote.go

```diff
@@ -387,6 +388,20 @@ func (md *MDServerRemote) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+// CheckReachability implements the MDServer interface.
+func (md *MDServerRemote) CheckReachability(ctx context.Context) {
```

Comment:
It would be less code, would avoid opening a new connection, and would test the actual connection in use, if you call `md.pinger.pingOnce()` instead of using `net.DialTimeout`.  I don't really see a downside, unless the reachability protocol is explicitly trying to measure _new_ connectivity to the server, rather than existing connectivity.